includes/admin/settings/class-wc-settings-products.php

```diff
@@ -386,6 +386,16 @@ protected function get_settings_for_downloadable_section() {
 					'autoload' => false,
 				),
 
+				array(
+					'desc'          => __( 'Allow using redirect mode as a last resort', 'woocommerce' ),
```

Comment:
Perhaps we can change the string to `redirect mode (insecure)` to be consistent with labelling above? wdyt?

Reply:
Updated :+1: